spec/connection_spec.rb

```diff
@@ -19,8 +19,21 @@
     webkit_pid.should be > 1
     read_io.close
     Process.kill(9, fork_pid)
-    sleep 1
-    expect { Process.getpgid(webkit_pid) }.to raise_error Errno::ESRCH
+    eventually{expect { Process.getpgid(webkit_pid) }.to raise_error Errno::ESRCH}
```

Comment:
Could you add a space around the `{` and `}` to  be consistent with existing formatting?